docs/guides/sap/btp-integration.md

```diff
@@ -0,0 +1,12 @@
+# SAP BTP integration
+
+The Camunda SAP BTP integration is an artifact run on BTP. It connects to Camunda 8 SaaS, and 
+
+- provides a generic Fiori app for starting BPMN processes and displaying Camunda Forms in the Fiori design language
+- exposes selected BTP services to BPMN tasks - and vice versa
+- provides a generic endpoint to start BPMN processes with
+
+> [!IMPORTANT]
+>
+> The BTP integration is in alpha phase available upon request - please use the https://camunda.com/contact to get in touch.
```

Comment:
Is this an alpha feature, or an experimental feature like RPA? See [this PR](https://github.com/camunda/camunda-docs/pull/4213) for context.